crates/storage/src/config.rs

```diff
@@ -0,0 +1,2 @@
+pub const REDB_FILE: &str = "ream.durability.redb";
```

Comment:
```suggestion
pub const REDB_FILE: &str = "ream.redb";
```

Why do we need the word durability